File: rangesum/range.java

	
	public static void main(String[] args)
    {
			Scanner sc=new Scanner(System.in);
            int n = sc.nextInt(); 
			int []arr= Arrays.stream(sc.next().split("[,\\[\\]]")).filter(w-> !w.equals("")).mapToInt(Integer::parseInt).toArray();
		    Solution g=new Solution();
            int k1 = sc.nextInt();
            int k2 = sc.nextInt();  
            int ans=g.rangeSum(arr,n,k1,k2);  
            System.out.println(ans);
          
	}
}
class Solution
{
	public static int rangeSum(int arr[],int n,int lower,int upper)
        {
        
			int count_sum =-1;
            //Write your code here without removing the existing code.
	        //the variable 'arr' contains array of integers of size n and lower and upper is an integer.
	        //modified the variable 'count_sum' contain the output of the program in integer format.
            return count_sum;
        }
}

File: rangesum/range.py
# Starter Code
class Solution:
    def rangeSum(self, arr, n,k):
        count_sum =-1
        #Write your code here without removing the existing code.
	    #the variable 'arr' contains array of integers of size n and lower and upper is an integer.
	    #modified the variable 'count_sum' contain the output of the program in integer format.
        return count_sum


if __name__ == '__main__':
    n = int(input())
    str = input()
    arr = [int(int(i)) for i in str.strip('[]').split(',')]
    x = Solution()
    k1= int(input())
    k2= int(input())
    ans = x.rangeSum(arr,n,k1,k2)
    print(ans)

